frontend/src/metabase/query_builder/components/saved-question-picker/SavedQuestionPicker.jsx

```diff
@@ -142,6 +142,7 @@ const mapStateToProps = ({ currentUser }) => ({ currentUser });
 export default _.compose(
   Collection.loadList({
     query: () => ({ tree: true }),
+    reload: true,
```

Comment:
I don't think it's quite possible to fix every `reducer` without force reloading collections. The reason is that it's almost impossible to do so without resorting to hacking a lot. The collection has a property called `here`, which I guess from how it looks.
![image](https://user-images.githubusercontent.com/1937582/159666100-8591ab28-af80-4e96-92c8-1ff949ff0fe0.png)
Tells whether a collection contains models or questions or not. To use only `reducer` in entities without resorting to using `reload: true` means we need a way to know that after we move a model/question from one collection to another, how should we change the `here` to as there could be multiple scenarios e.g.

- before moved: `here: ["card", "dataset"]`, after moved: we need to know which collection a question/model is moved from and what we should set `here` to. As it could be `["card"]`, `["dataset"]`, or even `["card", "dataset"]` depending on what's on that collection after a question/models is moved away.